examples/common/eventdisplay/FairEveMCTracks.cxx

```diff
@@ -151,9 +156,10 @@ InitStatus FairEveMCTracks::Init()
     FairField *field = ana->GetField();
     if (field == nullptr) {
         LOG(ERROR) << "Lack of magnetic field map!";
-        field = new FairField();
+        //        field = new FairField();
```

Comment:
Is this intended? If yes, let's remove it, otherwise a commented out line without any further comments will hurt readability of the code.

Reply:
One comment to @karabowi, are we sure that now magnetic field is present in all examples, fRK is used by  FairEveMCTracks::DrawTrack.